src-backend/registration/tests/test_login.py

```diff
@@ -0,0 +1,40 @@
+from django.test import TestCase, Client
+from django.contrib.auth.models import User
+from rest_framework.authtoken.models import Token
+import json
+
+
+class LoginTest(TestCase):
+    def setUp(self):
+        self.client = Client()
+
+    def test_invalid_method_cannot_login(self):
+        response = self.client.get('/auth/login')
+        self.assertEqual(response.status_code, 405)  # Method not allowed
```

Comment:
please use the nose test stuff we have set up in the other project (or open an issue to convert it to use it)